src/slack_annotations/format.py

```diff
@@ -85,9 +90,11 @@ def _build_annotation_summary(annotation: dict[str, Any]) -> str:
     else:
         document_link = uri
 
-    if display_name:
-        return f"`{username}` ({display_name}) annotated {document_link}:"
-    return f"`{username}` annotated {document_link}:"
+    formatted_user = (
+        f"`{username}` ({display_name})" if display_name else f"`{username}`"
+    )
+    formatted_group = f" in `{group_name}`" if group_name else ""
```

Comment:
Dan asked that these messages have more clear indication where they're coming from exactly.